homework_no_git/js/app.js

```diff
@@ -0,0 +1,116 @@
+
+// Section 1
+// 1. Parameter - Variable defined in the function's declaration when the function is defined.
+// // Argument - Actual values passed into the function when the function is called.
+
```

Comment:
your answer on this is the most accurate and precise of all submissions -- nice work